### PR TITLE
add ability to parse decimal

### PR DESCRIPTION
**Pull request by pikatenor**
🙅‍♂ 
```
JapaneseNumbers::Parser.parse('13.9万円')
=> 130000
```

🙆  
```
JapaneseNumbers::Parser.parse('13.9万円')
=> 139000
```

### CODE REVIEW

File: lib/japanese_numbers/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module JapaneseNumbers
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```
Comment (osyoyu) [inline lib/japanese_numbers/version.rb]: Can you remove this change? I prefer version bumps to be independent from pull requests

**Review by osyoyu — COMMENTED** [review]
thx :+1:

File: lib/japanese_numbers/version.rb (hunk 1 above)
Comment (pikatenor) [reply lib/japanese_numbers/version.rb]: okay

**Review by osyoyu — APPROVED** [review]

**Comment by osyoyu** [comment]
Released! https://rubygems.org/gems/japanese_numbers/versions/0.2.0

**Pull request merged**